.github/workflows/ci.yml

```
@@ -0,0 +1,62 @@
+name: Eradiate test
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  robot:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[doc,recommended,test,dev]
+          pip install eradiate-mitsuba
```

How about using the `kernel` extra to resolve all dependencies in a single pass?
```suggestion
          pip install .[kernel,doc,recommended,test,dev]
```

However, I think we are at risk of this failing if the tested branch uses an unreleased kernel version. Maybe we should hold this until the full CI pipeline is ready for roll out?

---

Should we actually merge a version that is built on a unreleased kernel? Shouldn't we let such resulting CI actions fail as a marker of their unmerged development status? Your formulation is more elegant

---

Only releases require a wheel; even `main` might have stable, but not yet released, kernel code. So I think it's legit to build the kernel as part of CI.

Anyway, I think it's more appropriate to split this PR between separate CI and testing tooling parts (basically, move the CI update commits to a separate branch) (see my general comment).

---

Moved to CI PR #482